### PR TITLE
Update readme python

### PR DESCRIPTION
**Pull request by Tony133**
Update Readme

### CODE REVIEW

File: PHP/README.md [hunk 1]
```diff
@@ -61,10 +61,17 @@ Go to https://www.apachefriends.org/index.html, download and install XAMPP.
 6. Open the browser and navigate to:
 	`http://localhost/hello-world.php`
 
-## OS X
+## Mac OS
 
 ### Installation
-1. Since apache and PHP come packaged with OS X no need to install manually again. 
+1. Since apache and PHP come packaged with Max OS no need to install manually again.
```
Comment (grrttedwards) [inline PHP/README.md]: There's a typo here "Max OS"

Comment (Tony133) [reply PHP/README.md]: I fixed 

thanks

Tony133

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Reference the Issues that you are solving this PR! I see that you have changes three different files! Make sure you reference those Issues!

**Comment by sbshah97** [comment]
Any problems here?

**Comment by sbshah97** [comment]
Any updates @Tony133

**Comment by Tony133** [comment]
Hello, sorry if I did not answer you, but I did not have time in these two days.
If you want to do the merge, if I think of adding more things I make a new pull request

**Review by sbshah97 — APPROVED** [review]

**Pull request merged**